application/common/models/Mfa.php

```diff
@@ -279,6 +285,10 @@ public static function create(int $userId, string $type, string $label = null):
             'status' => 'success',
         ]);
 
+
+        /* @todo uncomment this line when the emails are ready */
+        // self::sendAppropriateMessages($user, self::EVENT_TYPE_CREATE);
```

Comment:
This seems like it would be better placed in an `afterSave()` method on this `Mfa` model (which does include information about whether it was an insert or an update). That way, we know that this `sendAppropriateMessages(...)` will be called for each new Mfa, even if it was created outside of this static `Mfa::create(...)` method.

Reply:
Done